services/backend/services/v4/compose.base.yaml

```diff
@@ -7,6 +7,12 @@ services:
         condition: service_started
     volumes:
       - ./config/functionalAccounts.json:/home/node/app/functionalAccounts.json
+      - ../../../../entrypoints/loop_entrypoints.sh:/usr/local/bin/loop_entrypoints.sh
+      - ./entrypoints/db_migration.sh:/docker-entrypoints/0.sh
+    entrypoint:
+      - loop_entrypoints.sh
+      - docker-entrypoint.sh
```

Comment:
I think this is useless but I did the same way it's done in v3

Reply:
yes, I think as well. I just wanted to be safer to always apply the latest default `docker-entrypoint` from the container, in case they will do something more useful in the future (for now it just allows to avoid explicitly writing `node` in the command whenever you just want to add options

Reply:
How do multiple entrypoints work?

Reply:
they are executed one after the other, so, it would be `. loop_entrypoints.sh docker-entrypoint.sh`. One needs to implement the entrypoint then so that they can take another sh script as input. That's done with the exec "$@" bit which both entrypoints have